### ISSUE TITLE
Feature suggestion: Add "Commit and push" option on Mac client in same way as is done on Windows client

### ISSUE THREAD

**Issue by jaltin** [ForkIssues/Tracker#2227]
In the Windows client I have the option to Commit, but also the option to do "Commit and push" by pressing the "down arrow" next to the "Commit" button. See the screenshot for illustration:

![image](https://github.com/user-attachments/assets/febca4b9-ec6b-4a71-be29-88bdd1607019)

I like this feature as it allows me to do either "Commit" or "Commit and push" easily.

On the Mac client I only have the option "Commit". I know that there is an option to press alt to see the "Commit and push" button, but I don't always remember that exists.

Could you add the same functionality as is on Windows Client (as in the screenshot)?

**Comment by DanPristupov** [comment]
> I know that there is an option to press alt to see the "Commit and push" button, but I don't always remember that exists.

The 'option' key is a macOS feature. It's everywhere, on every menu, in fact.

**Comment by jaltin** [comment]
@DanPristupov  Yes I know, my issue posting was just a suggestion to make it easier for me as a user to see the option to do "Commit and push". I think it would be nice if the same feature that is in Windows client was there. It sure makes my cognitive load of using Fork lower if they behave the same where possible.

Thx
